### PR TITLE
14th batch

### PR DESCRIPTION
**Pull request by stevsmit**
Batch 14. 

Preview: https://deploy-preview-42423--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes#ocp-4-10-bug-fixes

### CODE REVIEW

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -1028,6 +1032,18 @@ This update corrects the accepted label specification so that the plug-in import
 
 * Previously, when pressing a pool inside the block pools page, the final `Ready` phase persisted after deletion. Consequently, the pool was in the `Ready` state even after deletion. This update redirects users to the *Pools* page and refreshes the pools after detention. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1981396[*BZ#1981396*])
 
+[discrete]
+[id="ocp-4-10-dns-bug-fixes"]
+==== DNS
+
+* Previously, the DNS Operator did not enable the `prometheus` pug-in in the server blocks for custom upstream resolvers. Consequently, CoreDNS did not report metrics for upstream resolvers and only reported metrics for the default server block. With this update, the DNS Operator was changed to enable the `prometheus` plug-in in all server blocks. CoreDNS now reprts Prometheus metrics for custom upstream resolvers. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2020489[*BZ#2020489*])
```
Comment (opayne1) [inline release_notes/ocp-4-10-release-notes.adoc:1039]: s/reprts/reports

Comment (opayne1) [reply release_notes/ocp-4-10-release-notes.adoc:1039]: should it be domain name system (DNS) or is that not really necessary here?

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -1028,6 +1032,18 @@ This update corrects the accepted label specification so that the plug-in import
...
+
+* Previously, an upstream DNS that provided a response greater than 512 characters caused an application to fail. This occurred because it could not clone the repository from GitHub because to DNS could not be resolved. With this update, the bufsize for KNI CoreDNS is set to 521 to avoid name resolutions from GitHub. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1991067[*BZ#1991067*])
```
Comment (opayne1) [inline release_notes/ocp-4-10-release-notes.adoc:1041]: should bufsize be in `` here?

File: release_notes/ocp-4-10-release-notes.adoc [hunk 3]
```diff
@@ -1028,6 +1032,18 @@ This update corrects the accepted label specification so that the plug-in import
...
...e Operator gets the cluster DNS service object from the API to determine whether the Operator needs to create or update the service. If the service already exists, the Operator compares it with what the Operator expects to get in order to determine whether an update is needed. Kubernetes 1.22, on which {product-title} 4.9 is based, introduced a new `spec.internalTrafficPolicy` API field for services. The Operator leaves this field empty when it creates the service, but the API sets a default value for this field. The Operator was observing this default value and trying to update the field back to the empty value. This caused the Operator's update logic to continuously try to revert the default value that the API set for the service's internal traffic policy. With this fix, when comparing services to determine whether an update is required, the Operator now treats the empty value and default value for `spec.internalTrafficPolicy` as equal. As a result, the Operator no longer spuriously tries to update the cluster DNS service when the API sets a default value for the service's `spec.internalTrafficPolicy field`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2002461[*BZ#2002461*])
```
Comment (opayne1) [inline release_notes/ocp-4-10-release-notes.adoc:1043]: s/in order to/to

Comment (opayne1) [reply release_notes/ocp-4-10-release-notes.adoc:1043]: Is there a way to break this down a little more? Just seems like a bit much, though I am not sure how you would really do that. I just tried to provide a suggestion but it seemed too wordy so idk where you could cut down.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 4]
```diff
@@ -1028,6 +1032,18 @@ This update corrects the accepted label specification so that the plug-in import
...
+
+* Previously, the DNS Operator did not enable the cache plugin for server blocks in the CoreDNS Corefile configuration map corresponding to entries in the `spec.servers` field of the `dnses.operator.openshift.io/default` object. As a result, CoreDNS did not cache responses from upstream resolvers that were configured using `spec.servers`. With this bug fix, the DNS Operator is changed to enable the cache plugin for all server blocks, using the same parameters that the Operator already configured for the default server block. CoreDNS now caches responses from all upstream resolvers. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2006803[*BZ#2006803*])
+
```
Comment (opayne1) [inline release_notes/ocp-4-10-release-notes.adoc:1046]: s/plugin/plug-in x2

Comment (opayne1) [reply release_notes/ocp-4-10-release-notes.adoc:1046]: should it be `Corefile`?

File: release_notes/ocp-4-10-release-notes.adoc [hunk 5]
```diff
@@ -1220,6 +1236,10 @@ This update corrects the accepted label specification so that the plug-in import
 
 * Previously, the cluster ingress router was blocked from working because the Ingress Operator failed to configure a wildcard DNS record for the cluster ingress router on Azure Stack Hub IPI. With this fix, the Ingress Operator now uses the configured ARM endpoint to configure DNS on Azure Stack Hub IPI. As a result, the cluster ingress router now works properly. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2032566[*BZ#2032566*])
 
+* Previously, the cluster-wide prox configuration could not accept IPv6 addresses for the `noProxy` setting. Consequently, it was impossible to install a cluster whose configuration was having `noProxy` with IPv6 addresses. With this update, the Cluster Network Operator is now able to parse IPv6 addresses for the `noProxy` setting of the cluster-wide proxy resource. As a result, it is now possible to exclude IPv6 addresses from proxying. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1939435[*BZ#1939435*])
```
Comment (opayne1) [inline release_notes/ocp-4-10-release-notes.adoc:1239]: Maybe:
s/proxying/for the `noProxy` setting

File: release_notes/ocp-4-10-release-notes.adoc [hunk 6]
```diff
@@ -1220,6 +1236,10 @@ This update corrects the accepted label specification so that the plug-in import
...
...he Ingress Operator set default values for these subfields when the Operator admitted or re-admitted an IngressController that specified the "HostNetwork" or "NodePortService" strategy type. With this bug, however, the Operator ignored updates to these spec fields, and updating `spec.endpointPublishingStrategy.hostNetwork.protocol` or `spec.endpointPublishingStrategy.nodePort.protocol` to `PROXY` to enable PROXY protocol on an existing IngressController had no effect. To work around this issue, it was necessary to delete and recreate the IngressController to enable PROXY protocol. With this update, the Ingress Operator is changed so that it correctly updates the status fields when `status.endpointPublishingStrategy.hostNetwork` and `status.endpointPublishingStrategy.nodePort` are null and when the IngressController spec fields specify PROXY protocol with the `HostNetwork` or `NodePortService` endpoint publishing strategy type. As a result, setting `spec.endpointPublishingStrategy.hostNetwork.protocol` or `spec.endpointPublishingStrategy.nodePort.protocol` to `PROXY`now takes proper effect on upgraded clusters. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1997226[*BZ#1997226*])
```
Comment (opayne1) [inline release_notes/ocp-4-10-release-notes.adoc:1241]: you have PROXY in backticks for some and some are not. Are they all supposed to have backticks or not have backticks?

**Review by opayne1 — COMMENTED** [review]
Just a few comments from me :)

File: release_notes/ocp-4-10-release-notes.adoc (hunk 6 above)
Comment (stevsmit) [reply release_notes/ocp-4-10-release-notes.adoc:1241]: I think they're just context dependent. Brandi wrote this one; but for example, when you set something to `proxy`, you would put it in backticks. When it's proxy protocol I guess it's a noun?
